hello-world-notification-microservice/demo/measurements.py

```diff
@@ -0,0 +1,101 @@
+import base64
+import json
+import sys
+from datetime import datetime
+from time import sleep
+
+from requests import Session
+from requests.adapters import HTTPAdapter
+
+_tenant_id = ''
+_username = 'admin'
+_password = ''
+_platform_url = ''
+
+
+def get_session(tenant_id=_tenant_id, username=_username, password=_password):
+    auth_string = f'{tenant_id}/{username}:{password}'
+    session = Session()
+    session.verify = False
+    session.headers.update({
+        'Content-Type': 'application/json',
+        'Accept': 'application/json',
+        'Authorization': 'Basic ' + base64.b64encode(auth_string.encode()).decode()
+    })
+    http_adapter = HTTPAdapter()
+    session.mount('https://', http_adapter)
+    session.mount('http://', http_adapter)
+    return session
+
+
+def post_create_device(session, url=_platform_url):
+    device_request = {
+        "name": "testMeasurementDevice",
+        "c8y_IsDevice": {},
+        "c8y_SupportedMeasurements": ["c8y_Speed"]
+    }
+    json_body_str = json.dumps(device_request)
+    return check_status_code(session.post(f'{url}/inventory/managedObjects', data=json_body_str), 201)
+
+
+def post_delete_device(session, device_id, url=_platform_url, ):
+    return check_status_code(session.delete(f'{url}/inventory/managedObjects/{device_id}'), 204)
+
+
+def post_measurement(measurement_str, session, url=_platform_url):
+    return check_status_code(session.post(f'{url}/measurement/measurements', data=measurement_str), 201)
+
+
+def create_measurement_str(device_id):
+    measurement = {
+        'c8y_SpeedMeasurement': {
+            'T': {
+                'value': 100,
+                'unit': 'km/h'
+            }
+        },
+        'time': datetime.now().strftime('%Y-%m-%dT%H:%M:%S.000+00:00'),
+        'source': {'id': str(device_id)},
+        'type': 'c8y_Speed'
+    }
+    return json.dumps(measurement)
+
+
+def check_status_code(_response, expected_code):
+    if _response.status_code != expected_code:
+        raise RuntimeError(f'Response code: {_response.status_code}', str(_response.json()))
+    else:
+        return _response
+
+
+if __name__ == '__main__':
+    args = sys.argv[1:]
+    if len(args) > 0:
+        command = args[0]
```

Comment:
should we print a simple message 'unknown option; use device, delete and create' if it is something other than one of the expected options.
We could maybe be a bit more careful with accessing args[]; if any are deficient of required options, we can print error message. Otherwise user will get ugly stack-trace